operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ Entries should be ordered as follows:
 Entries should include a reference to the Pull Request that introduced the change.
 
 ## main / unreleased
+* [CHANGE] **breaking change** Make `ConfigMap` the default for `configStorageType`. This means that the Mimir (or Enterprise Metrics) configuration is now created in and loaded from a ConfigMap instead of a Secret. #2277
+  - Set to `Secret` to keep existing way of working. See related #2031, #2017, #2089.
```

Comment:
Is there any guide on how to migrate to a ConfigMap?

Reply:
No, there's not :(